java/src/test/java/ai/rapids/cudf/BinaryOpTest.java

```diff
@@ -1072,16 +1072,6 @@ public void testShiftRightUnsigned() {
                (b, l, r, i) -> b.append(((long)(l.getInt(i) >>> r))))) {
         assertColumnsAreEqual(expected, answer, "int32 >>> scalar = int64");
       }
-
-      try (Scalar s = Scalar.fromShort((short) 0x0000FFFF);
-           ColumnVector answer = s.shiftRightUnsigned(shiftBy, DType.INT16);
-           ColumnVector expected = forEachS(DType.INT16, (short) 0x0000FFFF,  shiftBy,
```

Comment:
Curious why we're abandoning coverage of INT16 here rather than fixing the Java code to produce the correct value to compare against?  Java may not be able to unsigned right shift shorts directly, but that doesn't mean we can't compute the expected value in Java.

Reply:
We didn't have coverage for byte either, but it does not mean we need to exhaustively test everything.  The goal for most of these tests is to see if the plumbing is in place for JNI.  I would argue if we really want correctness tests those need to go in the C++ code, which was updated with the code was fixed.  If you really want me to put in hard coded results for a short shift I can do it, but I don't see that much value in doing it.

Reply:
It's an unsigned right shift.  There's nothing to hard-code here, just load the short value as an int (mask by 0xFFFF), right-shift the int, then cast back to short.

But if we don't want to bother, that's fine too.